srcpkgs/CBQN/template

```diff
@@ -0,0 +1,50 @@
+# Template file for 'CBQN'
+pkgname=CBQN
+version=0.2.0
+revision=1
+archs="x86_64*"
```

Comment:
Why is this restricted?

Reply:
CBQN will only compile correctly for x86_64-glibc and x86_64-musl.

Reply:
Yeah, the makefile needs to be fixed (or replaced) to work for cross-build.